PubnativeLite/PubnativeLite/Ad Request/VWAdFactory.m

```diff
@@ -39,7 +40,7 @@ - (VWAdRequestModel *)createVWAdRequestWithZoneID:(NSString *)zoneID withAdSize:
     
     [self setIDFA:adRequestModel];
     
-    if (![HyBidSettings sharedInstance].coppa) {
+    if (![HyBidSettings sharedInstance].coppa && [[HyBidUserDataManager sharedInstance] canCollectData] && ![[VWAdLibrary shared] usPrivacyOptOut]) {
         adRequestModel.requestParameters[@"age"] = [[HyBidSettings sharedInstance].targeting.age stringValue];
```

Comment:
You should check if us privacy string isn't null also. 
usPrivacyOptout will assume user didn't opted out if string is null. 
Check VWAdFactory for reference

Reply:
@faresbh fixed. check my latest commit.